haiku-apps/fakbetur/fakbetur-1.0.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="An accounting application for Haiku"
+DESCRIPTION="FakBEtur, an accounting application that includes invoice editing."
+APP="FakBEtur"
+HOMEPAGE="https://github.com/ModeenF/beos-fakBEtur"
+COPYRIGHT="2014 Maciej Witkowiak"
+LICENSE="MIT"
+REVISION="1"
+COMMIT="536da5c9c687586e7ade59152b9466820b773fb6"
+SOURCE_URI="$HOMEPAGE/archive/$COMMIT.tar.gz"
+CHECKSUM_SHA256="f38207db028896d6cc66736f081830d473755cd996bdd16512e5ac63c20e7d35"
+SOURCE_DIR="beos-fakBEtur-$COMMIT"
```

Comment:
It might be worth replacing COMMIT by srcGitRev in this recipe, as most of the recipes which use a git revision define srcGitRev:

variable name | srcGitRev | COMMIT | gitSrcRevision
------------------ | ------------ | ------------ | ------------------
\# occurences |      111       |       28      |          4

(We're not going to change what is already done, but for new recipes I think it would be nice to stick to what has "almost" become a "standard" :)
Would it be OK for you?
Thanks!

Everything else looks OK :)